lib/ecto/repo/supervisor.ex

```diff
@@ -154,6 +154,29 @@ defmodule Ecto.Repo.Supervisor do
     end
   end
 
+  @compile {:inline, triplet: 2, maybe_put_stacktrace: 2}
```

Comment:
Since the `triplet` function is not used in this module the `inline` annotation has no effect - it only potentially inlines the function inside of the module